sky/execution.py

```diff
@@ -402,6 +414,12 @@ def exec(  # pylint: disable=redefined-builtin
         ValueError: if the specified cluster does not exist or is not in UP
             status.
     """
+    entrypoint = task
+    if isinstance(entrypoint, sky.Dag):
+        logger.warning(
+            f'{colorama.Fore.YELLOW}Use a sky.Dag as an entrypoint in '
```

Comment:
nit: Passing a sky.Dag to sky.exec() is deprecated. Pass a sky.Task instead.

Reply:
Done. Thanks!